### PR TITLE
Changes to issuance_txid_digest to align with ZIP 244

### PR DESCRIPTION
**Pull request by vivek-arte**

### CODE REVIEW

File: zip-0227.rst [hunk 1]
```diff
@@ -291,10 +291,13 @@ The following is a list of rationale for different decisions made in the proposa
...
-A new issuance transaction digest algorithm is defined that constructs the identifier for an issuance transaction. Each branch of the tree of hashes will correspond to a specific subset of transaction data. The overall structure of the hash is as follows; each name referenced here will be described in detail below::
+
+As in ZIP 244 [#zip-0244]_, the digests are all personalized BLAKE2b-256 hashes, and in cases where no elements are available for hashing, a personalized hash of the empty byte array is used.
+
+A new issuance transaction digest algorithm is defined that constructs the identifier for an issuance transaction. Each branch of the tree of hashes will correspond to a specific subset of issuance transaction data. The overall structure of the hash is as follows; each name referenced here will be described in detail below::
 
     issuance_txid_digest
-    ├── issueActions
+    ├── issue_actions_digest
     └── issuerVerificationKey
```
Comment (PaulLaux) [inline zip-0227.rst:301]: They both should have the same case: `issue_actions` or `issueActions` not both.

File: zip-0227.rst [hunk 2]
```diff
@@ -303,36 +306,46 @@ issuance_txid_digest
 --------------------
 A BLAKE2b-256 hash of the following values ::
 
-   T.1: issueActions            (field encoding bytes)
-   T.2: issuerVerificationKey   (field encoding bytes)
+   T.1: issue_actions_digest    (32-byte hash output)
+   T.2: issuanceValidatingKey   (32 bytes)
```
Comment (PaulLaux) [inline zip-0227.rst:310]: Same for naming convention.

**Review by PaulLaux — APPROVED** [review]
Overall good, added minor comments.

File: zip-0227.rst (hunk 1 above)
Comment (vivek-arte) [reply zip-0227.rst:301]: The naming convention in ZIP 244 is at the bottom of [this](https://zips.z.cash/zip-0244#txid-digest) section -- `camelCase` if it is not a hash of something else, `snake_case`

I see an inconsistency in the use of both `issuerVerificationKey` and `issuanceValidatingKey` which I am now fixing.

**Pull request merged**